lib/src/Uncertainty/Distribution/openturns/HistogramFactory.hxx

```diff
@@ -50,12 +50,18 @@ public:
                      const Scalar bandwidth) const;
   Distribution build(const Sample & sample,
                      const UnsignedInteger binNumber) const;
+  Distribution build(const Sample & sample,
+                     const Scalar first,
+                     const Point width) const;
```

Comment:
Here width should be passed by reference to avoid costly copy

Reply:
Thanks for the information: I missed that "&" also provided a performance advantage at this point.